backends/platform/sdl/win32/win32.cpp

```diff
@@ -253,6 +253,23 @@ Common::String OSystem_Win32::getSystemLanguage() const {
 	return OSystem_SDL::getSystemLanguage();
 }
 
+Common::String OSystem_Win32::getDefaultIconPath() {
+	TCHAR iconsPath[MAX_PATH];
+
+	if (_isPortable) {
+		Win32::getProcessDirectory(iconsPath, MAX_PATH);
```

Comment:
Does it mean it will stick all icons next to scummvm.exe? What about adding here a small sanity check: you try to see if `gui-icons.dat` file is present in the local directory or in the `icons/` subdirectory.

Reply:
This was an oversight; I now have it in `Icons/`.

I don't know that we need an extra check; the whole point is that this is the default path. If the user wants to put it at the root directory or somewhere else, they certainly can.